conformance/utils/kubernetes/helpers.go

```diff
@@ -64,10 +64,10 @@ func NewGatewayRef(nn types.NamespacedName, listenerNames ...string) GatewayRef
 }
 
 // GWCMustBeAccepted waits until the specified GatewayClass has an Accepted
-// condition set to true. It also returns the ControllerName for the
-// GatewayClass. This will cause the test to halt if the specified timeout is
-// exceeded.
-func GWCMustBeAccepted(t *testing.T, c client.Client, timeoutConfig config.TimeoutConfig, gwcName string) string {
+// condition set. Passing an empty status string means that any Status will do.
```

Comment:
I think this language may be a little bit more clear:

```suggestion
// condition set. Passing an empty status string means that any value will be accepted.
```